src/Config/PrefetchOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Extensions.RabbitMQ
+{
+    public class PrefetchOptions
```

Comment:
Let's make RabbitMQOptions implement IOptionsFormatter which all our other extension options classes implement, so the configured options are logged. That will help us diagnose issues, particularly now that with your changes the options allow for more interesting behavior changes.